vochain/process.go

```diff
@@ -66,9 +66,13 @@ func (v *State) CancelProcess(pid []byte) error { // LEGACY
 func (v *State) Process(pid []byte, isQuery bool) (*models.Process, error) {
 	var processBytes []byte
 	var err error
-	v.RLock()
+	if !isQuery {
+		v.Tx.RLock()
+	}
 	processBytes, err = v.mainTreeViewer(isQuery).DeepGet(pid, ProcessesCfg)
-	v.RUnlock()
+	if !isQuery {
```

Comment:
The same second if here, having the same condition, seems strange to me. Can we do ?
```golang
if !isQuery { 
    v.Tx.RLock()
    defer v.Tx.RUnlock()
}
```